core/src/main/scala/org/apache/spark/util/ThreadUtils.scala

```diff
@@ -183,15 +182,15 @@ private[spark] object ThreadUtils {
   /**
    * Construct a new Scala ForkJoinPool with a specified max parallelism and name prefix.
    */
-  def newForkJoinPool(prefix: String, maxThreadNumber: Int): SForkJoinPool = {
+  def newForkJoinPool(prefix: String, maxThreadNumber: Int): java.util.concurrent.ForkJoinPool = {
     // Custom factory to set thread names
-    val factory = new SForkJoinPool.ForkJoinWorkerThreadFactory {
-      override def newThread(pool: SForkJoinPool) =
-        new SForkJoinWorkerThread(pool) {
+    val factory = new java.util.concurrent.ForkJoinPool.ForkJoinWorkerThreadFactory {
```

Comment:
Import the class rather then fully qualify it 5 times?

Reply:
Updated. Thanks！